pylxd/models/container.py

```diff
@@ -84,6 +84,12 @@ def put(self, filepath, data):
                 params={'path': filepath}, data=data)
             return response.status_code == 200
 
+        def delete(self, filepath):
+            response = self._client.api.containers[
```

Comment:
We probably want to put a test in as to whether the feature is supported.

e.g.

```python
if u'file_delete' not in self._client.host_info['api_extensions']:
    raise some error that makes sense here.
```

On one of my test boxes (16.04 LTS for example), my lxd is 2.0.8 which is missing that feature.  This would save a round-trip to the server when it's not going to work.

See  https://github.com/lxc/lxd/blob/master/doc/api-extensions.md and https://github.com/lxc/lxd/blob/master/doc/rest-api.md#delete-pathpathinsidethecontainer for more details.

Reply:
Aha, that explains why it was missing.